hudi-platform-service/hudi-table-service-manager/src/main/java/org/apache/hudi/table/service/manager/common/CommandConfig.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.table.service.manager.common;
+
+import com.beust.jcommander.Parameter;
+
+public class CommandConfig {
+  @Parameter(names = {"--server-port", "-p"}, description = "Server Port")
+  public Integer serverPort = 9092;
+
+  @Parameter(names = {"-schedule-interval-ms"}, description = "Schedule Interval Ms")
+  public Long scheduleIntervalMs = 30000L;
+
+  @Parameter(names = {"-schedule-core-executor-size"}, description = "Schedule Core Execute Size")
+  public Integer scheduleCoreExecuteSize = 300;
+
+  @Parameter(names = {"-schedule-max-executor-size"}, description = "Schedule Max Execute Size")
+  public Integer scheduleMaxExecuteSize = 1000;
+
+  @Parameter(names = {"-metadata-store-class"}, description = "Metadata Store Class")
+  public String metadataStoreClass = "org.apache.hudi.table.service.manager.store.impl.RelationDBBasedStore";
+
+  @Parameter(names = {"-instance-cache-enable"}, description = "Instance Cache Enable")
+  public boolean instanceCacheEnable = true;
+
+  @Parameter(names = {"-instance-max-retry-num"}, description = "Instance Max Retry Num")
+  public Integer instanceMaxRetryNum = 3;
+
+  @Parameter(names = {"-instance-submit-timeout-sec"}, description = "Instance Submit Timeout Sec")
+  public Integer instanceSubmitTimeoutSec = 600;
+
+  @Parameter(names = {"-spark-submit-jar-path"}, description = "Spark Submit Jar Path")
```

Comment:
this is spark-specific? it's under `common/CommandConfig` so it better be engine agnostic.